testsuite/tests/kuadrant/authorino/authorization/opa/test_authorization_services.py

```diff
@@ -0,0 +1,184 @@
+    Auth for user who owns the protected resource, a.k.a. "owner" user.
+    The "uma_protection" client role is assigned to the user so that they are allowed to create protected resources.
+    """
+    owner = rhsso.realm.create_user("owner", "owner")
+    role = rhsso.realm.admin.get_client_role(client_id=rhsso.client.client_id, role_name="uma_protection")
+    rhsso.realm.admin.assign_client_role(user_id=owner.user_id, client_id=rhsso.client.client_id, roles=[role])
+    return HttpxOidcClientAuth.from_user(rhsso.get_token(owner.username, owner.password), owner)
+
+
+@pytest.fixture(scope="module")
+def requester_auth(rhsso):
+    """Auth for user who requests the access to the protected resource, a.k.a. "requester" user"""
+    requester = rhsso.realm.create_user("requester", "requester")
+    return HttpxOidcClientAuth.from_user(rhsso.get_token(requester.username, requester.password), requester)
+
+
+@pytest.fixture(scope="module")
+def owner_uma(rhsso, resource_owner_auth):
+    """UMA client used to create a protected resource and assign permissions for "requester" to access it."""
+    keycloak_connection = KeycloakOpenIDConnection(
+        server_url=rhsso.server_url,
+        client_id=rhsso.client_name,
+        client_secret_key=rhsso.client.secret,
+        username=resource_owner_auth.username,
+        password=resource_owner_auth.password,
+        realm_name=rhsso.realm_name,
+    )
+    return KeycloakUMA(keycloak_connection)
+
+
+@pytest.fixture(scope="module")
+def protected_resource(owner_uma, resource_owner_auth):
+    """
+    Protected resource created by and owned by "owner" user
+    """
+    resource_representation = owner_uma.resource_set_create(
+        payload={
+            "name": "anything-1",
+            "uris": ["/anything/1"],
+            "owner": resource_owner_auth.username,
+            "ownerManagedAccess": "true",
+            "scopes": ["get", "post"],
+        }
+    )
+    return resource_representation
+
+
+def test_user_managed_access(client, resource_owner_auth, requester_auth, protected_resource, owner_uma):
+    """Tests that UMA authorization flow works as expected."""
+
+    # Access the protected resource by requester
+    response = client.get("/anything/1", auth=requester_auth)
+    assert response.status_code == 403
+
+    # Access the protected resource by the resource owner is forbidden since there are no permissions configured
+    # The mere ownership is not sufficient to access it
+    response = client.get("/anything/1", auth=resource_owner_auth)
+    assert response.status_code == 403
+
+    # Allow HTTP GET access to the protected resource for "requester" user
+    # Only resource owner is allowed to do this
+    owner_uma.policy_resource_create(
+        protected_resource["_id"],
+        {
+            "name": "Allow GET for requester",
+            "description": "Allow GET for requester",
+            "scopes": ["get"],
+            "users": [f"{requester_auth.username}"],
```

Comment:
You can use `[requester_auth.username]`

Reply:
nice, will update as suggested

Reply:
updated and force-pushed as suggested, could you please re-review (this was the only change in force-push)